.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+#### What did you change? \*
+
+<!--- Describe your changes in detail. -->
+
+#### Why? \*
```

Comment:
What do you think about merging this section with the first one?

Reply:
I would rather keep it so people can be more objective.

Reply:
I think it would make it easier to read and to write, the less topics to fill the better. So, one could describe the changes and embed the reason for them, seems more fluid to me

Reply:
Disagree, people are lazy and tend to don't give enough info on their PRs. That's why I think specifically asking for it in an objective field will force them to complete the info needed. And as you saw, I've marked some fields as required, so that would be one reason to deny the PR.

Reply:
Makes sense